### PR TITLE
Add contributor profile for pig208.

### PR DESCRIPTION
**Pull request by PIG208**
This is a follow-up for the merged PR wechaty@python-wechaty#253.

Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by lijiarui** [comment]
Hi @PIG208 thanks for adding the profile, please:

1. make Github action green
2. add more info on the profile page
3. add your PR content to the profile page

File: jekyll/_contributors/pig208.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+site: https://github.com/PIG208
+avatar: /assets/contributors/pig208/avatar.webp
+bio: 学生开发者，纽约大学在读学生，前后端都沾点
+email: p359101898@gmail.com
+twitter: 
+---
+
+近期主要在使用Python, 为python-wechaty贡献代码。同样有用过Node.js, TypeScript, etc.
+希望在开源项目中保证代码规范性。
+
```
Comment (lijiarui) [inline jekyll/_contributors/pig208.md:16]: Please add your PR link here.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/pig208.md (hunk 1 above)
Comment (PIG208) [reply jekyll/_contributors/pig208.md:16]: Updated accordingly.

Comment (lijiarui) [reply jekyll/_contributors/pig208.md:16]:
```
[为python-wechaty贡献代码](https://github.com/wechaty/python-wechaty/pull/253) --->

为python-wechaty贡献代码, 贡献内容:
- [Implement mention_text, contact-bot, and improve typing configuration](https://github.com/wechaty/python-wechaty/pull/253)。
```

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Please make Github action green, thanks!

**Comment by PIG208** [comment]
Sorry for the delay, I was on a flight. This PR should be all-set now.

**Pull request merged**